PowerShell/JumpCloud Module/Private/settings/New-JCSettingsFile.ps1

```diff
@@ -0,0 +1,63 @@
+function New-JCSettingsFile {
+    [CmdletBinding()]
+    param (
+        [Parameter(
+            HelpMessage = 'To Force Re-Creation of the Config file, set the $force parameter to $tru'
+        )]
+        [switch]
+        $force
+    )
+
+    begin {
+        # Config should be in /PowerShell/JumpCloudModule/Config.json
+        $ModuleRoot = (Get-Item -Path:($PSScriptRoot)).Parent.Parent.FullName
+        $configFilePath = join-path -path $ModuleRoot -childpath 'Config.json'
+
+        # Define Default Settings for the Config file
+        $config = @{
+            'moduleBanner' = @{
+                'Message'      = @{value = 'JumpCloud PowerShell Module now processes Get requests in parallel, to disable this functionailty run: Set-JCSettingsFile -parallelOverride $true'; write = $false; copy = $false };
```

Comment:
Will this have to be manually updated each time or is this dynamically set from ModuleBanner.md?